docs/solutions/Hadoop/EL7/Cloudera/Development/scripts/master_setup.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+## Installation script for primary master node in cluster
+## by Zachary Smith (Zachary.Smith@oracle.com)
+## Last Update - March 2018
+
```

Comment:
Would be useful to have some high level comments in these scripts to explain what they're doing.

Reply:
Adding comments - will commit shortly.